pkg/utils/notify/notify.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notify
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	securejoin "github.com/cyphar/filepath-securejoin"
+	"github.com/s3rj1k/go-fanotify/fanotify"
+	"golang.org/x/sys/unix"
+)
+
+// RuntimePaths is the list of paths where the container runtime runc or crun
+// could be installed. Depending on the Linux distribution, it could be in
+// different locations.
+//
+// When this package is executed in a container, it prepends the HOST_ROOT env
+// variable to the path.
+var RuntimePaths = []string{
+	"/bin/runc",
+	"/usr/bin/runc",
+	"/usr/sbin/runc",
+	"/usr/local/bin/runc",
+	"/usr/local/sbin/runc",
+	"/usr/lib/cri-o-runc/sbin/runc",
+	"/run/torcx/unpack/docker/bin/runc", // Used in TODO
```

Comment:
```suggestion
	"/run/torcx/unpack/docker/bin/runc", // Used in Flatcar Container Linux
```

Reply:
Actually, https://www.flatcar.org/blog/2024/04/os-innovation-with-systemd-sysext/ says:

> Therefore, we recently removed Torcx and recommend using systemd-sysext for deploying custom Docker/containerd versions.

But we can keep the torcx path for a while for compatibility with older Flatcar versions.

@pothos When using systemd-sysext to install `runc`, `crun` or `conmon`, will it be in `/usr/bin/` or do we need to add another path here?

Reply:
Should be in `/usr/bin/`: https://stable.release.flatcar-linux.net/amd64-usr/current/rootfs-included-sysexts/containerd-flatcar_contents.txt https://stable.release.flatcar-linux.net/amd64-usr/current/rootfs-included-sysexts/docker-flatcar_contents.txt